nslsii/iocs/tests/test_epstwostate_ioc.py

```diff
@@ -0,0 +1,99 @@
+import subprocess
+import os
+import sys
+import time
+
+from ophyd import Device, EpicsSignal, EpicsSignalRO
+from ophyd.device import (Component as Cpt,
+                          FormattedComponent as FmtCpt)
+
+
+class EPSTwoStateDevice(Device):
+
+    def __init__(self, *, prefix, name, **kwargs):
+        super().__init__(prefix=prefix, name=name, **kwargs)
+
+    status = Cpt(EpicsSignalRO, 'Pos-Sts', string=True)
+
+    opn_cmd = FmtCpt(EpicsSignal,
+                     '{self.prefix}Cmd:Opn-Cmd',
+                     string=True, kind='omitted')
+    cls_cmd = FmtCpt(EpicsSignal,
+                     '{self.prefix}Cmd:Cls-Cmd',
+                     string=True, kind='omitted')
+
+
+def test_epstwostate_ioc():
```

Comment:
We should probably add tests of the other two 'failure modes', or include them as a part of this test.